packages/adapter/addon/rest.js

```diff
@@ -1057,7 +1057,15 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
   },
 
   _fetchRequest(options) {
-    return fetch(options.url, options);
+    let fetchFunction = fetch();
+
+    if (fetchFunction) {
+      return fetchFunction(options.url, options);
+    } else {
+      throw new Error(
+        'cannot find the `fetch` module or the `fetch` global. Did you mean to install the `ember-fetch` addon?'
```

Comment:
Do we need the same error in both places? What does that buy us?

Reply:
I wasn't sure which of the two we should do. At first glance, I'd prefer it to happen at the top level.